cli/create.go

```diff
@@ -29,7 +29,12 @@ const PresetNone = "none"
 
 var ErrNoPresetFound = xerrors.New("no preset found")
 
-func (r *RootCmd) create() *serpent.Command {
+type CreateOptions struct {
```

Comment:
Maybe you should put a comment here explaining what these are for. E.g. "CreateOptions provides middleware for the workspace creation command so it can be extended for other create commands."